Kernel/CMakeLists.txt

```diff
@@ -402,6 +403,8 @@ if ("${SERENITY_ARCH}" STREQUAL "x86_64")
 
         Arch/x86_64/VGA/IOArbiter.cpp
 
+        Arch/x86_64/PowerState.cpp
+
```

Comment:
Maybe lose this newline, you didn't add it for aarch64 so let's not add it here.